hikari/impl/stateful_event_manager.py

```diff
@@ -200,7 +203,12 @@ async def on_guild_ban_remove(self, shard: gateway_shard.GatewayShard, payload:
 
     async def on_guild_emojis_update(self, shard: gateway_shard.GatewayShard, payload: data_binding.JSONObject) -> None:
         """See https://discord.com/developers/docs/topics/gateway#guild-emojis-update for more info."""
-        event = self._app.event_factory.deserialize_guild_emojis_update_event(shard, payload)
+        old = []
+        for emoji_id in self._cache.get_emojis_view_for_guild(snowflakes.Snowflake(payload["guild_id"])):
+            if emoji := self._cache.get_emoji(emoji_id):
+                old.append(emoji)
```

Comment:
This is really inecessary this could simply be this:
```suggestion
        emojis_view = self._cache.get_emojis_view_for_guild(snowflakes.Snowflake(payload["guild_id"]))
        old = list(emojis_view.values())
```